### DISCUSSION TITLE
[FEATURE]: Add "VariationImages" to getListings "includes" parameter

### DISCUSSION THREAD

**Discussion by creharmony** [etsy/open-api#717] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

I'm using V3 API (instead of V2) to retrieve listings and listings having variations

- recently 'inventory' has been added to the `includes` of [getListing](https://developers.etsy.com/documentation/reference/#operation/getListing) with #236 🙏  thanks for that; this prevent me additional api call.

- about variations translations:
- - I'm always using an additional `getListingInventory` to retrieve 'fr' translations of variations **property_names** 
- - I also do need variation `property_values.values` translation : that is traced (& commented april) in #431 

And I created this ticket to know if you plan to add to includes "**variationImages**' too ?  this is the way to retrieve  the relation between : a product property_id, a product property_value_id, and an image_id

for now I'm using [`getListingVariationImages`](https://developers.etsy.com/documentation/reference/#operation/getListingVariationImages)

### Describe the solution you'd like

having `variationImages` accepted as `includes` enum value.

This will add to the result an addotionnal field ; example
```
<listingDocHere>
...
variationImages: [
    {
      "property_id": 102868018123,
      "value_id": 112744446756,
      "value": "Quartz",
      "image_id": 3215768148
    },
    {
      "property_id": 102868018123,
      "value_id": 112394957206,
      "value": "Améthyste",
      "image_id": 3215768258
    },
    {
      "property_id": 102868018123,
      "value_id": 1014722082046,
      "value": "Labradorite",
      "image_id": 3215768376
    }
  ]
```` 

### Describe any alternatives you've considered

As alternative I will keep my additional api call to getListingVariationImages

**Comment by etsyachristensen** [comment]
I will pass this to the team as a ticket to see if it's something that we will take on or not.

**Comment by etsyachristensen** [comment]
We'll convert this ticket to a new discussion in the Ideas category to continue tracking the request.
